src/renderer/components/NodeSelectorPanel/TextBox.tsx

```diff
@@ -4,7 +4,7 @@ import { MouseEventHandler, memo } from 'react';
 export interface TextBoxProps {
     text: string;
     collapsed?: boolean;
-    toolTip?: React.ReactNode | string;
+    toolTip?: React.ReactNode | string | string;
```

Comment:
Did you mean to add another string instead of removing the existing one?

Reply:
Lol, I was testing out an ESLint rule and forgot to remove this.